rpcserver.go

```diff
@@ -5063,13 +5070,16 @@ func handleSendRawTransaction(s *rpcServer, cmd interface{}, closeChan <-chan st
 		// JSON-RPC error is returned to the client with the
 		// deserialization error code (to match bitcoind behavior).
 		if _, ok := err.(mempool.RuleError); ok {
-			rpcsLog.Debugf("Rejected transaction %v: %v", tx.Hash(),
+			err = fmt.Errorf("Rejected transaction %v: %v", tx.Hash(),
 				err)
-		} else {
-			rpcsLog.Errorf("Failed to process transaction %v: %v",
-				tx.Hash(), err)
+			rpcsLog.Debugf(err.Error(), "")
```

Comment:
All of these should use "%v" or "%s" as the format string, not the error string.